css/_uswds-theme-custom-styles.scss

```diff
@@ -2710,13 +2710,13 @@ $theme-table-column-gap: 4;
     }
   }
 
-  h3 {
+  &__header_second {
     @include typeset("lang", "lg", 2);
     color: color("base-dark");
     font-weight: fw("thin");
   }
 
-  h4 {
+  &__header_third {
```

Comment:
We should use BEM naming convention here since it's not Jekyll/Liquid.

Consider using something like `header--primary, header--secondary, header--tertiary`. 

_We could also try something like `.h1, .h2, .h3`, but this might have a negative impact on a11y/semantics._

Reply:
I updated the class names to BEM in [1d5e163](https://github.com/uswds/uswds-site/pull/2608/commits/1d5e163cbb0b5ed91f5f4c846e0a111c33038012). I also updated the template vars to match the primary/secondary/tertiary naming structure.